services/api/service.go

```diff
@@ -333,6 +336,9 @@ func (api *RelayAPI) StopServer() (err error) {
 
 		// wait a few seconds, for any pending getPayload call to complete
 		time.Sleep(5 * time.Second)
```

Comment:
It's used because a proposer might just have called getHeader, and the relay shuts down before getPayload is called. This delay gives the proposer a little time to get the getPayload call to the relay webserver.